tests/chart_tests/test_byo_sa.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+from tests import supported_k8s_versions
+from tests.chart_tests.helm_template_generator import render_chart
+
+
+@pytest.mark.parametrize(
+    "kube_version",
+    supported_k8s_versions,
+)
+class TestServiceAccounts:
+    def test_serviceaccount_rbac_disabled(self, kube_version):
+        # Render the chart with rbacEnabled set to False
```

Comment:
Unnecessary comment, we can see in the next two lines that rbacEnabled is False.
```suggestion
```